sdk/identity/identity/src/credentials/clientCertificateCredential.ts

```diff
@@ -59,135 +30,18 @@ export class ClientCertificateCredential implements TokenCredential {
     tenantId: string,
     clientId: string,
     certificatePath: string,
-    options?: ClientCertificateCredentialOptions
+    options: ClientCertificateCredentialOptions = {}
```

Comment:
Why are we changing this from optional to required with a default?

Reply:
It's not required by default! This end syntax ` = {}` says that it's optional, and that by default we will make a plain object internally.